### DISCUSSION TITLE
Minimal Database Backups

### DISCUSSION THREAD

**Discussion by joshreed13** [immich-app/immich#11408] [Q&A] [answered]
Hey, thanks for the great app! I'm looking for advice about how to handle backups. The docs at https://immich.app/docs/administration/backup-and-restore/ explain how the filesystem separates original content from generated content so that it can be backed up separately. This makes it easy to save a minimal back up of Immich's data.

My question is if there is anything similar for the database? I would like to back up the part of the database that holds content original to Immich (users, albums, favorites, etc) without backing up data that can be regenerated (exif metadata, face recognition, smart search, etc). I would appreciate any guidance here. Thanks!

**Comment by mmomjian** [comment]
This currently isn’t possible. I suppose it would be, but it seems like a lot of work to save, what, a few hundred MB max on the backup?

**Comment by mertalev** [comment, marked as answer]
The amount of time and IO it would take to regenerate the ML data etc. would outweigh whatever you gain in storage, honestly.

However, skipping the `geodata_places` table can be a good idea since the data in it is just from a geodata dump and gets regenerated every release anyway.

**Reply by joshreed13** [reply]
Thanks. I figured the savings weren't much, but I'd like to follow the best practices. And thanks for the heads up about `geodata_places`. It's good to know about anything that changes over time even if there's no new content. That could accumulate in historical backups over time.
